src/Apps/Artist/Components/StyledLink.tsx

```diff
@@ -0,0 +1,12 @@
+import { RouterLink } from "Artsy/Router/RouterLink"
```

Comment:
This file was pulled out because importing it from the `ArtistApp` directly causes the `ArtistApp` to be bundled with the app shell. We'll have to use a linter or other mechanism to  help people avoid this pitfall.